pkg/cmd/accept/exec.go

```diff
@@ -24,11 +24,13 @@ import (
 const (
 	groupNameBootstrap               = "system:bootstrappers:managedcluster"
 	userNameSignatureBootstrapPrefix = "system:bootstrap:"
-	userNameSignatureSA              = "system:serviceaccount:open-cluster-management:cluster-bootstrap"
+	userNameSignatureSAPrefix         = "system:serviceaccount:"
 	groupNameSA                      = "system:serviceaccounts:open-cluster-management"
+	groupNameSAManualToken           = "system:serviceaccounts:default"
```

Comment:
this means we are trying an sa in default ns, I do not think it is a common case.